src/build/help/help.xml

```diff
@@ -2112,6 +2112,8 @@
                     <p>The restore command automatically defaults to selecting the latest backup from the first repository where backups exist (see <link page="user-guide" section="/quickstart/perform-restore">Quick Start - Restore a Backup</link>). The order in which the repositories are checked is dictated by the <file>pgbackrest.conf</file> (e.g. repo1 will be checked before repo2). To select from a specific repository, the <br-option>{[dash]}-repo</br-option> option can be passed (e.g. <br-option>{[dash]}-repo=1</br-option>). The <br-option>{[dash]}-set</br-option> option can be passed if a backup other than the latest is desired.</p>
 
                     <p>When PITR of <br-option>{[dash]}-type=time</br-option> is specified, then the target time must be specified with the <br-option>{[dash]}-target</br-option> option. If a backup is not specified via the <br-option>{[dash]}-set</br-option> option, then the configured repositories will be checked, in order, for a backup that contains the requested time. If no backup can be found, the latest backup from the first repository containing backups will be used. For other types of PITR, e.g. <id>xid</id>, the <br-option>{[dash]}-set</br-option> option must be provided if the target is prior to the latest backup. See <link page="user-guide" section="/pitr">Point-in-Time Recovery</link> for more details and examples.</p>
+
+                    <p>Replication slots are not included per recommendation of the <postgres/> documentation. See the <postgres/> documentation: <link url="https://www.postgresql.org/docs/current/continuous-archiving.html#BACKUP-LOWLEVEL-BASE-BACKUP">Making a Base Backup Using the Low Level API</link> (subsection Backing Up The Data Directory) for more information.</p>
```

Comment:
Perhaps it would be better to link directly to the subsection we want (#BACKUP-LOWLEVEL-BASE-BACKUP-DATA)?

Reply:
resolved 476854f9b5fc043976d9bc6f387ddaa87fcbe79c 
Thanks, I had looked for but failed to determine how to link that subsection.

Reply:
I do it by searching for the section title in the page source, then finding the link nearby.